src/Symfony/Component/Security/Core/Authorization/AuthorizationChecker.php

```diff
@@ -57,6 +57,10 @@ public function __construct(TokenStorageInterface $tokenStorage, AuthenticationM
             $attributes = [$attributes];
         }
 
+        if (\count($attributes) > 1) {
+            @trigger_error('Passing more than one Security attribute to '.__METHOD__.' is deprecated since Symfony 4.4. Use multiple isGranted() calls or the expression language (e.g. "has_role(...) or has_role(...)") instead.', \E_USER_DEPRECATED);
```

Comment:
Could we completely deprecate passing arrays to `isGranted` instead of allowing passing arrays with only one element ?

Reply:
Agreed, thanks!